build.gradle

```diff
@@ -16,6 +17,7 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath 'com.github.ben-manes:gradle-versions-plugin:0.17.0'
         classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.11'
+        classpath "org.apache.logging.log4j:log4j-api:$log4JVersion"
```

Comment:
Not here. `classpath` refers to the gradle system itself, not the code you're compiling.